mimic/model/valkyrie_objects.py

```diff
@@ -0,0 +1,140 @@
+"""
+Model objects for the Valkyrie mimic.
+"""
+from characteristic import attributes, Attribute
+from uuid import uuid4
+from json import loads, dumps
+
+from mimic.util.helper import random_hex_generator
+
+class AccountContactPermission(object):
+    """
+    An intersection object representing a certain contact's permissions on a certain account's items
+    """
+
+    permission_type_map = {
```

Comment:
Each item in the map appears to be sequentially identified.  Is there a reason why you didn't select a list for this map?  E.g.,
```python
permission_type_map = [
    "view_ticket",
    "edit_ticket",
    "view_community",
    ...etc...
]
```